src/GtkUtilities.jl

```diff
@@ -44,7 +44,7 @@ include("panzoom.jl")
 using .PanZoom
 import .PanZoom: interior, fullview   # for extensions
 
-function Base.copy!{C<:Colorant}(ctx::CairoContext, img::AbstractArray{C})
+function Base.copy!{C<:Union{Colorant,Number}}(ctx::CairoContext, img::AbstractArray{C})
```

Comment:
It's inarguably more convenient this way, the issue is the global nature of module method tables and the undetected dependency problem. If some dependency of a dependency happens to use GtkUtilities (or any other example of this), then you may happily start using this method without realizing where it was actually defined, or properly declaring your own dependency on it. When some other subcomponent refactors and stops using this package, totally unrelated code could break.